repose-aggregator/src/docs/asciidoc/filters/keystone-v2-authorization.adoc

```diff
@@ -0,0 +1,204 @@
+= Keystone v2 Authorization Filter
+
+The Keystone v2 Authorization Filter authorizes requests based on data about the user making the request.
+
+== General filter information
+* *Name:* keystone-v2-authorization
+* *Default Configuration:* keystone-v2-authorization.cfg.xml
+* *Released:* TODO
+* *Bundle:* repose-filter-bundle
+* link:../schemas/keystone-v2-authorization.xsd[Schema]
+
+== Prerequisites & Postconditions
+=== Required Request Headers
+This filter does not require any request headers.
+
+=== Required Preceding Filters
+This filter requires that the <<keystone-v2.adoc#, Keystone v2 Filter>> precede it to populate necessary user data.
+
+=== Request Headers Created
+The following headers are created:
+
+* `X-Tenant-Id` - The Tenant ID's for the user.
+  This header will only be added if this filter is configured to validate tenants.
```

Comment:
Didn't we just say yesterday it'll almost always do at least one?

Reply:
The authentication filter will, but this filter only messes with the tenant header if tenant authorization is performed. What would it do otherwise? Theoretically, the authentication filter has already written the header with appropriate values.